docs/content/guides/dagster/software-defined-assets.mdx

```diff
@@ -58,26 +58,30 @@ def hottest_dates(daily_temperature_highs: DataFrame) -> DataFrame:
 
 The framework infers asset dependencies by looking at the names of the arguments to the decorated functions. E.g. the function that defines the `daily_temperature_highs` asset has an argument named `sfo_q2_weather_sample` - corresponding to the asset of the same name.
 
-### Building a job
+### Combining the assets into a group
 
-Having defined some assets, we can use <PyObject module="dagster.core.asset_defs" object="build_assets_job" /> to build a job that, when it runs, materializes them - i.e. it computes their contents and writes them to a location in persistent storage.
+Having defined some assets, we can combine them into an <PyObject object="AssetGroup" />, which allows working with them in Dagit. It also allows combining them with resources and IO managers that determine how they're stored and connect them to external services.
```

Comment:
nit: maybe want to mention executor here, not sure if that's information overload.